docs/zh_cn/getting-started/for_distributed.md

```diff
@@ -135,21 +135,17 @@ JuiceFS 进程需要具有读写 `--cache-dir` 目录的权限。
 
 #### 开机自动挂载
 
-以 Linux 系统为例，假设客户端位于 `/usr/local/bin` 目录。将 JuiceFS 客户端重命名为 `mount.juicefs` 并复制到 `/sbin` 目录：
-
-```shell
-sudo cp /usr/local/bin/juicefs /sbin/mount.juicefs
-```
-
-编辑 `/etc/fstab` 配置文件，遵照 fstab 的规则添加一条新记录：
-
-```
-redis://tom:mypassword@myjfs-sh-abc.redis.rds.aliyuncs.com:6379/1    /mnt/myjfs    juicefs    _netdev,max-uploads=50,writeback,cache-size=512000     0  0
+可以用 `juiefs mount --update-fstab` 直接设置出自动挂载，例如：
```

Comment:
Need to prompt users that this is an unreleased feature and supported operating system. And the "Command Reference" documentation is missing a description of the `--update-fstab` option.